Sources/TuistGenerator/Generator/BuildPhaseGenerator.swift

```diff
@@ -126,6 +130,21 @@ final class BuildPhaseGenerator: BuildPhaseGenerating {
         }
     }
 
+    func generateScripts(_ scripts: [TargetScript],
+                         pbxTarget: PBXTarget,
+                         pbxproj: PBXProj)
+    {
+        scripts.forEach { script in
+            let buildPhase = PBXShellScriptBuildPhase(files: [],
+                                                      name: script.name,
+                                                      shellPath: "/bin/sh",
+                                                      shellScript: script.script,
+                                                      showEnvVarsInLog: true)
```

Comment:
minor: may be worth including `TargetScript.showEnvVarsInLog` for consistency with `TargetAction.showEnvVarsInLog` - optionally the `TargetActionEnvironmentMapper` could also set this too as does for `TargetAction`

Reply:
That's a good idea. I'll update it.